src/sonic-yang-models/yang-models/sonic-restapi.yang

```diff
@@ -0,0 +1,63 @@
+module sonic-restapi {
+
+    yang-version 1.1;
+
+    namespace "http://github.com/Azure/sonic-restapi";
+    prefix restapi;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    organization
+        "SONiC";
+
+    contact
+        "SONiC";
+
+    description "RESTAPI YANG Module for SONiC OS";
+
+    revision 2022-10-05 {
+        description "First Revision";
+    }
+
+    container sonic-restapi {
+
+        container RESTAPI {
+
+            description "RESTAPI TABLE part of config_db.json";
+
+            container certs {
+
+                leaf ca_crt {
+                    type string {
+                        pattern '(/[a-zA-Z0-9_-]+)*/([a-zA-Z0-9_-]+).pem';
+                    }
+                    description "Local path for ca_crt.";
+                }
+
+                leaf server_crt {
+                    type string {
+                        pattern '(/[a-zA-Z0-9_-]+)*/([a-zA-Z0-9_-]+).crt';
+                    }
+                    description "Local path for server_crt.";
+                }
+
+                leaf client_crt_cname {
+                    type string {
+                        pattern '(/[a-zA-Z0-9_-.]+)*/([a-zA-Z0-9_-.]+)./([a-z]{3})';
```

Comment:
>([a-z]{3}) [](http://example.com/codeflow?start=72&length=10)

What is the issue in original PR? Do you add a new test case for the issue?

Reply:
it was a syntax error in regex. i've added few more tests to cover.